ecmascript/transforms/base/src/hygiene/mod.rs

```diff
@@ -665,6 +665,25 @@ impl<'a> VisitMut for Hygiene<'a> {
         n.class.visit_mut_with(self);
     }
 
+    fn visit_mut_class_method(&mut self, n: &mut ClassMethod) {
+        let mut folder = Hygiene {
+            config: self.config.clone(),
+            current: Scope::new(ScopeKind::Fn, None),
+            ident_type: IdentType::Ref,
+            var_kind: None,
+        };
+
+        folder.ident_type = IdentType::Binding;
+        n.function.params.visit_mut_children_with(&mut folder);
+
+        folder.ident_type = IdentType::Ref;
+
+        n.function.body.visit_mut_with(self);
```

Comment:
I think `decorators ` should be also visited. Maybe it can be visited at first, before creating `folder`.

Reply:
Thanks for the catch. Applied changes as suggested.